src/app/mystudy/applied/page.tsx

```diff
@@ -36,24 +37,8 @@ export default function StudyAppliedPage() {
           <div className="bg-white px-7 py-3 rounded-t-[20px]">내가 지원한 스터디</div>
         </div>
         <div className=" w-[950px] bg-white flex flex-row flex-wrap justify-center items-center rounded-[40px] pb-[30px] mb-[40px] gap-x-[40px]">
-          {studyData.length > 0 &&
-            studyData.map((data) => (
-              <StudyAppliedCard
-                key={data.id}
-                id={data.id}
-                field={data.field}
-                detailField={data.detailField}
-                title={data.title}
-                content={data.content}
-                minMemberCount={data.minMemberCount}
-                nowMemberCount={data.nowMemberCount}
-                level={data.level}
-                isInPerson={data.isInPerson}
-                location={data.location}
-                isClosed={data.isClosed}
-              />
-            ))}
-          {studyData.length === 0 || (studyData.length % 2 === 1 && <StudyAppliedEmptyDataCard />)}
+          {data.length > 0 && data.map((data) => <StudyAppliedCard key={data.id} studyData={data} />)}
+          {data.length === 0 || (data.length % 2 === 1 && <StudyAppliedEmptyDataCard />)}
```

Comment:
```suggestion
          {data.length && data.map((data) => <StudyAppliedCard key={data.id} studyData={data} />)}
          {!data.length || (data.length % 2 === 1 && <StudyAppliedEmptyDataCard />)}
```
으로 하시는 것도 가능할 것 같아요!!

Reply:
그렇네요! 수정하였습니다 
[fix: 문법 수정](https://github.com/mobi-community/mobi-ttak-kong/pull/18/commits/ff7025eb3c585408e025400ff002069c03d0fc3f)